src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/debugImageDetails/candidate/utils.tsx

```diff
@@ -51,7 +51,9 @@ export function getSourceTooltipDescription(
   builtinSymbolSources: Array<BuiltinSymbolSource> | null
 ) {
   if (source === INTERNAL_SOURCE) {
-    return t('This debug information file was uploaded via Sentry CLI.');
+    return t(
+      "This debug information file is from Sentry's internal symbol server for this project"
```

Comment:
```suggestion
      "This debug information file is from Sentry's internal symbol server for this project."
```

Not sure if this should have a period on it.

Reply:
The other tooltips don't have it, so I will remove it...thanks for pointing that